ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -290,7 +290,7 @@ evalFunctionCall
     ;
 
 highlightFunction
-    : HIGHLIGHT LT_PRTHS field=relevanceField  RT_PRTHS            #highlightFunctionCall
+    : HIGHLIGHT LT_PRTHS field=relevanceField (COMMA relevanceArg)* RT_PRTHS #highlightFunctionCall
```

Comment:
I think we shouldn't use there `relevanceArgs`.
The following query isn't rejected:
```sql
SELECT highlight('Tags', boost=42) ...
```